DependencyInjection/SncRedisExtension.php

```diff
@@ -150,9 +150,7 @@ protected function loadClient(array $client, ContainerBuilder $container)
      */
     protected function loadPredisClient(array $client, ContainerBuilder $container)
     {
-        if (null === $client['options']['cluster']) {
-            unset($client['options']['cluster']);
```

Comment:
Is this change tested? Because I am not sure if it will break Predis behavior or not.